Changes.md

```diff
@@ -28,6 +28,7 @@ API
 
 - EditScopeAlgo : Added support for editing attributes.
 - TweakPlug : Added `Create` mode.
+- VectorDataWidget : Added `dataMenuSignal` for constructing the context menu. Derived classes can add their own slots to this signal to modify the context menu and should use it in favor of `_createRows()`.
```

Comment:
- Anyone (not just derived classes) can use the signal, which is the benefit of the signal over the method.
- It's `_contextMenuDefinition()` that we're replacing, not `_createRows()`.